index.html

```diff
@@ -0,0 +1,559 @@
+                        <div class="latest-img-6"></div>
+                    </section>
+                    <ul class="latest-ul">
+                        <li><a href="#">PSA: Amazon Fresh grocery deliveries are now free for US Prime members</a></li>
+                        <li><a href="#">Alexis Ohanian and Serj Tankian team up to help build a social network for Armenia</a></li>
+                        <li><a href="#">Google Assistant gets its own phone number in India</a></li>
+                        <li><a href="#">Pam and Angela from The Office are re-watching the show for their new podcast</a></li>
+                        <li><a href="#">Bethesda wants you to test its game streaming tech with DOOM on iOS this year</a></li>
+                    </ul>
+                </div>
+            </section>
+            <section class="distract-latest-news">
+                <h2 class="apps-latest-title"><a href="#">Distract</a></h2>
+                <section class="distract">
+                    <h5 class="apps-most-popular">Most Popular</h5>
+                    <h3 class="apps-news-header"><a href="#">A massive cache of stolen OnlyFans videos have been dumped online</a></h3>
+                    <ul class="apps-news-ul">
+                        <li><a href="#">Rachel Kaiser</a></li>
+                        <li><time datetime="6h">6 hours </time>ago</li>
+                    </ul>
+                </section>
+                <div class="latest-news-second">
+                    <section class="latest-container-header">
+                        <header class="latest-news-header">
+                            <h5 class="latest-h5">latest</h5>
+                            <h4 class="latest-h4"><a href="#">How to find the secret surfing game in Microsoft Edge</a></h4>
+                        </header>
+                        <div class="latest-img-7"></div>
+                    </section>
+                    <ul class="latest-ul">
+                        <li><a href="#">Apple won't let villains have iPhones in movies, Star Wars director says</a></li>
+                        <li><a href="#">This Giphy/Jif peanut butter collab over GIF pronunciation is a no-brainer</a></li>
+                        <li><a href="#">Netflix added a top 10 list of its most-watched content — here’s how to find it</a></li>
+                        <li><a href="#">Hasbro announces 7 new D&D games starting with Baldur's Gate 3</a></li>
+                        <li><a href="#">Rumor: Call of Duty to launch free 'Warzone' battle royale game as early as February 18</a></li>
+                    </ul>
+                </div>
+            </section> 
+        </div>
+        <section class="latest-deals">
+            <header class="latest-deals-header">
+                <h2 class="latest-deals-left latest-deals-title"><a href="#">Latest deals</a></h2>
+                <a class="latest-deals-right" href="#">See all Deals</a>
+            </header>
```

Comment:
- [ ] In the medium and large breakpoints your images are losing proportion. They should be a little taller.
![Screenshot from 2020-03-16 11-57-49](https://user-images.githubusercontent.com/51926937/76786805-76355100-677d-11ea-8b34-cac193daecfb.png)

- [ ] In the small breakpoint, your images should be smaller (the same as the latest news section)
![Screenshot from 2020-03-16 12-01-35](https://user-images.githubusercontent.com/51926937/76787064-ee9c1200-677d-11ea-991d-9bc5e51905a7.png)

Should look like this:
![Screenshot from 2020-03-16 12-01-43](https://user-images.githubusercontent.com/51926937/76787072-f360c600-677d-11ea-9624-e9b1b375e117.png)